go/batch-submitter/drivers/sequencer/batch.go

```diff
@@ -54,14 +54,15 @@ func BatchElementFromBlock(block *l2types.Block) BatchElement {
 	isSequencerTx := tx.QueueOrigin() == l2types.QueueOriginSequencer
 
 	// Only include sequencer txs in the returned BatchElement.
-	if !isSequencerTx {
-		tx = nil
+	var cachedTx *CachedTx
+	if isSequencerTx {
+		cachedTx = NewCachedTx(tx)
```

Comment:
Technically the raw RLP bytes are on the `TxMeta` but its good to not use it so that we can delete it from the response/db. The JS batch submitter uses it. Its existence is an artifact of when transactions used a custom batch serialization